flink-formats/flink-csv/src/main/java/org/apache/flink/formats/csv/CsvCommons.java

```diff
@@ -35,6 +35,7 @@
 import static org.apache.flink.formats.csv.CsvFormatOptions.IGNORE_PARSE_ERRORS;
 import static org.apache.flink.formats.csv.CsvFormatOptions.NULL_LITERAL;
 import static org.apache.flink.formats.csv.CsvFormatOptions.QUOTE_CHARACTER;
+import static org.apache.flink.formats.csv.CsvFormatOptions.WRITE_BIGDECIMAL_AS_PLAIN;
```

Comment:
@afedulov or @PatrickRen any opinions about this solution implementation?

Reply:
I'm not quite familiar with Jackson but I think we need a test at least. @sunshineJK Could you add a test case to validate this feature?

Reply:
Okay, I'll add that test case later

Reply:
hello，I submitted the test case